flake.nix

```diff
@@ -20,6 +20,11 @@
 
       packages = forAllSystems (system: {
         home-manager = nixpkgsFor.${system}.callPackage ./home-manager { };
+        docs = with import ./docs { pkgs = nixpkgsFor.${system}; }; {
```

Comment:
I don't know much about flakes but I've understood that the `packages` attribute should be flat? I.e., should this introduce `docs-html`, `docs-manpages`, and `docs-json` as sibling attributes of `home-manager`?

Reply:
Technically, nixpkgs has sets like `vimPlugins`, `gnome` and many others and they can all be used from the command line:`nix run nixpkgs#gnome.rhythmbox`

Reply:
I believe that is because nixpkgs uses `legacyPackages`, but yes `packages` should be a flattened attribute set (which is why `flattenTree` is part of flake-utils).

`legacyPackages` does not add any constraint about the "shape" of its value.

Reply:
Thanks for the reviews!
I've now moved the packages out of the `docs` set, can you take another look?